.gitignore

```diff
@@ -1,5 +1,4 @@
-### https://raw.github.com/github/gitignore/f2ce448f2ba7a092da05482ceca99209127c0884/maven.gitignore
-
+# maven
```

Comment:
I'm not sure we want to change the `.gitignore` file. I would think these entries would be in one's global `.gitignore` file as the project itself does not generate these files